.gitmodules

```diff
@@ -62,6 +62,9 @@
 [submodule "s3select"]
 	path = src/s3select
 	url = https://github.com/ceph/s3select.git
+[submodule "src/cpp_redis"]
+	path = src/cpp_redis
+	url = https://github.com/alimaredia/cpp_redis.git
```

Comment:
As mentioned before this repo needs to be forked to Ceph. I'm adding this comment as a place holder until it is.